exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -193,13 +194,14 @@ def _create_database_info(self, db_ip_address: str, reused: bool) -> DatabaseInf
                 network_info=self.network_info,
                 volume_name=self._get_db_volume_name(),
             )
-            ssh_info = SshInfo(self.ssh_user, self.ssh_port_forward, self.ssh_key_file)
+            ssh_info = SshInfo(self.ssh_user, self.ssh_key_file)
             database_info = DatabaseInfo(
                 host=db_ip_address,
-                ports=Ports.default_ports,
+                ports=self.internal_ports,
                 reused=reused,
                 container_info=container_info,
                 ssh_info=ssh_info,
+                port_forwards=self.port_forwards,
```

Comment:
the parameter port_forwards expects a object of type Ports, however you stored a dictionary in self.port_forwards

Reply:
Uh-oh. Thank you 1000 times. I think I mixed up two concepts thoroughly. 
Will be fixed in next push.